examples/speech_to_text_demo/english_counter_pipeline.py

```diff
@@ -0,0 +1,10 @@
+from simuleval.agents import AgentPipeline
+from examples.demo.silero_vad import SileroVADAgent
```

Comment:
Good question, I thought about that also but I think it would be best to avoid adding dependencies on e.g. fairseq (for the S2S agent) inside the simuleval package. In any case I think users will only use these dummy agents as a template for writing their own agents, they shouldn't need to import them